src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -334,66 +395,98 @@ internal bool TryUnparse(StringBuilder sb)
             }
         }
 
-        public class ArgumentsCore
+        /// <summary>
+        /// The settings for <see cref="TextLoader"/>
+        /// </summary>
+        public class Options
         {
+            /// <summary>
+            /// Whether the input may include quoted values, which can contain separator characters, colons,
+            /// and distinguish empty values from missing values. When true, consecutive separators denote a
+            /// missing value and an empty value is denoted by \"\". When false, consecutive separators denote an empty value.
```

Comment:
>\"\" [](start = 63, length = 4)

where should be specific documentation magic to properly represent such strings. `<value></value> maybe?

Reply:
See my comment, but let me know if you think otherwise.

---
In reply to: [255806299](https://github.com/dotnet/machinelearning/pull/2417#discussion_r255806299) [](ancestors = 255806299)